source/acquaman/AMScanConfiguration.h

```diff
@@ -277,6 +293,10 @@ public slots:
 	  * with the axes of the scan.
 	  */
 	void setAxisControlInfos(const AMControlInfoList &axisControlInfos);
+	/// Sets the automatic direction assessment flag.
+	void setAutomaticDirectionAssessment(bool isAutomatic);
+	/// Sets the direction.
+	void setDirection(Direction newDirection);
```

Comment:
@helfrij there are two implementations for `setDirection()`, one of it is waiting for int (Ln97), the other is waiting for Direction